src/commands/build.ts

```diff
@@ -44,7 +54,9 @@ export default defineCommand({
         logLevel: ctx.args.logLevel as 'silent' | 'info' | 'verbose',
         // TODO: remove in 3.8
```

Comment:
Shall we remove it now we have 3.8?

Reply:
CLI is backward compatible i guess it would be harmless to keep and remove comment to mention why we have it extra here.